docs/source/what-blaze-isnt.rst

```diff
@@ -0,0 +1,71 @@
+=====================
+What Blaze Doesn't Do
+=====================
+
+Blaze suffers from over-hype.  The terms *Big-Data* and *Pandas* inevitably
+conflate in people's minds to become something unattainable and lead to
+disappointment.  Blaze is limited;  learning those limitations can direct you
+to greater productivity.
+
+First and foremost, Blaze does not replace Pandas.  Pandas will always be more
+feature rich and more mature than Blaze.  There are things that you simply
+can't do if you want to generalize out of memory.
+
+If your data fits nicely in memory then use NumPy/Pandas.  Your data probably
+fits nicely in memory.
+
+
+Some concrete things Blaze doesn't do
+-------------------------------------
+
+1.  Clean unstructured data - That petabyte of HTML you downloaded?  Can't
+    help you.
+2.  Most things in SciPy - You'd like to do gradient descent or run an FFT?  Out of luck.
+3.  Most things in SciKit Learn/Image/... - Ohhh, machine learning on terabytes of data!  Look elsewhere.
+4.  Statistical inference - We invite you to build this (this one is actually pretty doable.)
+5.  Parallelize your existing Python code - Not gonna happen.
+6.  Replace Spark - Blaze may operate on top of Spark, it doesn't compete with it.
```

Comment:
Maybe mention Hadoop here as well since that's nearly synonymous with big data. Maybe just say spark/Hadoop instead of only spark.